frontend/src/components/authentication/CreateAccountForm.jsx

```diff
@@ -0,0 +1,310 @@
+                  style={styles.inputs}
+                  onChange={this.passwordValidation}
+                />
+                {!isValidPassword && !isFirstPasswordLoad && (
+                  <div>
+                    <p style={styles.validationError}>
+                      {LOCALIZE.homePage.createAccount.content.inputs.passwordErrors.description}
+                    </p>
+                    <ul style={styles.validationError}>
+                      <li>
+                        {LOCALIZE.homePage.createAccount.content.inputs.passwordErrors.Bullet1}
+                      </li>
+                      <li>
+                        {LOCALIZE.homePage.createAccount.content.inputs.passwordErrors.Bullet2}
+                      </li>
+                      <li>
+                        {LOCALIZE.homePage.createAccount.content.inputs.passwordErrors.Bullet3}
+                      </li>
+                      <li>
+                        {LOCALIZE.homePage.createAccount.content.inputs.passwordErrors.Bullet4}
+                      </li>
+                      <li>
+                        {LOCALIZE.homePage.createAccount.content.inputs.passwordErrors.Bullet5}
+                      </li>
+                    </ul>
+                  </div>
+                )}
+              </div>
+              <div>
+                <div style={styles.inputTitle}>
+                  <span>
+                    {LOCALIZE.homePage.createAccount.content.inputs.passwordConfirmationTitle}
+                  </span>
+                </div>
+                {isValidPasswordConfirmation && (
+                  <span className="far fa-check-circle" style={styles.iconForOtherFields} />
+                )}
+                <input
+                  aria-label={
+                    LOCALIZE.homePage.createAccount.content.inputs.passwordConfirmationTitle
+                  }
+                  className={passwordConfirmationClassStyle}
+                  type="password"
+                  placeholder={
+                    LOCALIZE.homePage.createAccount.content.inputs.passwordConfirmationPlaceholder
+                  }
+                  id="password-confirmation"
+                  style={styles.inputs}
+                  onChange={this.passwordValidation}
+                />
+                {!isValidPasswordConfirmation && !isFirstPasswordLoad && (
+                  <p style={styles.validationError}>
+                    {LOCALIZE.homePage.createAccount.content.inputs.passwordConfirmationError}
+                  </p>
+                )}
+              </div>
+              <button
+                disabled={SUBMIT_BTN_DISABLED}
+                style={styles.loginBtn}
+                className="btn btn-primary"
+                type="submit"
+              >
+                {LOCALIZE.homePage.createAccount.button}
+              </button>
+            </form>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default LoginForm;
```

Comment:
Name doesn't match here either.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/152/commits/c69b9a2f0170c5941645d863a72f8391adf869cb